src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,112 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import java.util.Comparator;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+        Arrays.sort(persons, Comparator.comparingInt(Person::getAge));
```

Comment:
As I understand the exercise, here you should declare ```class``` that ```implements Comparator<Person>``` and use it's instance.